src/worker-type.js

```diff
@@ -499,6 +499,7 @@ WorkerType.createLaunchSpec = function(bid, worker, keyPrefix, provisionerId, pr
     price: bid.truePrice,
     launchSpecGenerated: new Date().toISOString(),
     workerModified: worker.lastModified.toISOString(),
+    lastModified: worker.lastModified.toISOString(),
```

Comment:
This is because I'm not sure if anyone was using workerModified and it's not that much extra space.